src/core/constants.js

```diff
@@ -15,14 +15,28 @@ export const VERSION = 'VERSION_WILL_BE_REPLACED_BY_BUILD';
 
 // GRAPHICS RENDERER
 /**
- * The default, two-dimensional renderer.
- * @typedef {unique symbol} P2D
+ * The default, two-dimensional renderer in p5.js.
+ *
+ * Use this when calling {@link p5.createCanvas|createCanvas} (for example,
+ * `createCanvas(400, 400, P2D)`) to specify a 2D context.
+ *
+ * @typedef {'p2d'} P2D
  * @property {P2D} P2D
  * @final
  */
 export const P2D = 'p2d';
 
-export const P2DHDR = 'p2d-hdr';
```

Comment:
I think you'll need to add this line back after your comment block to fix the tests -- looks like the error is that this constant is no longer defined.